src/grisp_cryptoauth_cert.erl

```diff
@@ -375,3 +385,22 @@ create_date_vars({generalTime, [Y1,Y2,Y3,Y4,M1,M2,D1,D2,H1,H2,48,48,48,48,90]})
     Day =   list_to_integer([D1,D2]),
     Hour =  list_to_integer([H1,H2]),
     {Year, Month, Day, Hour}.
+
+attribute_type('id-at-name') -> ?'id-at-name';
```

Comment:
I think it's way more user friendly to use the typical abbreviations here, something like e.g. `CN` for the common name.

Reply:
Not sure which data type to use for those abbreviations though, maybe `"DN"` is good enough.